setup.py

```diff
@@ -15,10 +15,14 @@
     'Click>=6.0',
     'boto3',
     'requests',
-    'hvac',
+    'hvac==0.6.0',
     # TODO: put package requirements here
 ]
 
+dependency_links=[
+    "git+ssh://git@github.com/stratasan/hvac.git@0.6.0#egg=hvac-0.6.0"
```

Comment:
Most installs won't have ssh creds to github though right? should this be `git+https://`?

Reply:
good point

Reply:
@sburns this repo is public, wont need creds